build/illume.js

```diff
@@ -11,17 +11,15 @@ var _q = require("@standard-library/q");
 var _kefir = require("kefir");
 
 var offsetAbove = function offsetAbove(y, element) {
-  return y > $(element).offset().top;
+  return y > element.offsetTop;
```

Comment:
The current jQuery code for this function is actually pretty pithy—though I'm not sure what browsers this is supposed to support: https://github.com/jquery/jquery/blob/1d2df772b4d6e5dbf91df6e75f4a1809f7879ab0/src/offset.js#L92-L106

These line might be possible to use in a function that accepts the `elem` argument and returns the top/left object.